templates/recover.html

```diff
@@ -1,14 +1,18 @@
 {% extends "auth.html" %}
 
 {% block main %}
-<div class="w-full max-w-xl text-center mx-auto">
-  <form class="js-validated-form bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4" onsubmit="event.preventDefault (); hedyApp.auth.submit ('recover')">
+<div class="w-full max-w-xl text-center mx-auto bg-white shadow-md rounded px-8 pt-6 pb-4">
+  <form class="js-validated-form " onsubmit="event.preventDefault (); hedyApp.auth.submit ('recover')">
     <h2 class="py-2">{{_('recover_password')}}</h2>
     <div class="flex flex-row items-center justify-center mb-2">
         <label class="inline-block w-1/3" for="username">{{_('username')}} / {{_('email')}}</label>
         <input class="inline-block w-1/2 mx-2 px-2 py-1 rounded-lg border-gray-400" id="username" name="username" required>
     </div>
     <button type="submit" class="green-btn my-2">{{_('send_password_recovery')}}</button>
   </form>
+  <div class="mt-4 border-t-2 border-gray-400 pt-4">
+      <h2>Reset wachtwoord met sleutel</h2>
+      <button class="green-btn my-2" onclick="window.location = '/reset_key'">Reset met sleutel</button>
+  </div>
```

Comment:
The content is hardcoded in Dutch. Not sure if this is something we want to address now or in a later PR.

Reply:
Forgot about that one! We first have to decide if we want to use the secret key approach, I agree that children will not write it down or simply forget where they wrote it. For the mail validation we use the html `valid` attribute, will take look if we can fix this so it only returns when also a `.nl` for example is added at the end of the mail.

Reply:
Temporary hidden this one